### ISSUE TITLE
As a rev, did not pass out after revs were defeated

### ISSUE THREAD

**Issue by the-orange-cow** [tgstation/tgstation#81235]
Round ID: [223213](https://scrubby.melonmesa.com/round/223213)
BYOND Version: 515.1626
Server revision compiled on: 
Master commit: [943f38bf7c5f9c048cc785deb0c537d57ee6ba77](https://www.github.com/tgstation/tgstation/commit/943f38bf7c5f9c048cc785deb0c537d57ee6ba77)
The following pull requests are currently test merged:
[#81089: 'Temp keybinds patch for 515.1628' by MrStonedOne at commit 45649ea7a5](https://www.github.com/tgstation/tgstation/pull/81089)
[#81120: 'Comprehensive cleanup of storage datum, replaces the weakrefs with just refs (because they were managed already)' by MrMelbert at commit bb98f3e02a](https://www.github.com/tgstation/tgstation/pull/81120)

Local commit: 97bde235abf11a67c68b2db76a092a0d613bcbd1
TGS version: /tg/station 13 Server v3.2.6.2
DMAPI version: 3.2.1.3

As a rev, after all the revheads were killed, none of the revs passed out during the mass deconversion.   This is kinda bad as several people didn't notice the chat announcement and continued to fight.

**Comment by Das15** [comment]
Apparently its still broken... Saw it happen on round [232008](https://scrubby.melonmesa.com/round/232008). It also didn't knock revs who were deconverted by a hit to the head.

**Comment by Jewelry-x** [comment]
After consulting with someone more knowledgeable, they claim that this was never the case. Upon reviewing the code, I found no evidence of revs passing out after de-converting, so there doesn't appear to be a bug in that area. I'll mark this issue for clean-up in case anything changes, but everyone I asked did not claim that this behavior was part of the system.

**Comment by the-orange-cow** [comment]
> After consulting with someone more knowledgeable, they claim that this was never the case. Upon reviewing the code, I found no evidence of revs passing out after de-converting, so there doesn't appear to be a bug in that area. I'll mark this issue for clean-up in case anything changes, but everyone I asked did not claim that this behavior was part of the system.

There absolutely was a mass rev pass out after all the revheads died and revs lost.  I can't remember if they passed out from a normal deconverting(I.e. windshield/head trauma).

**Comment by Jewelry-x** [comment]
> There absolutely was a mass rev pass out after all the revheads died and revs lost. I can't remember if they passed out from a normal deconverting(I.e. windshield/head trauma).

Well there is currently, as much as I am aware, no bug regarding the behaviour. The only reasonable explanation is that there was a PR about removing this mechanic, so I think it was intentional. Probably. I will have a look later to see what commits happened to the file. I will write another comment about it.

**Comment by the-orange-cow** [comment]
Trying to remember, I might have noticed the lack of knockout once or twice before I made the issue report.  But if I had to guess, the mass knockout was probably working in Q3 or Q4 2023.  I asked on discord, and mrmelbert remembered the knockout and said it used to be intentional.  He then pointed out that https://github.com/tgstation/tgstation/pull/76728 may have removed the knockout without documenting and recommended re-adding it as a bugfix.
